src/Apps/Order/Routes/__tests__/Accept.test.tsx

```diff
@@ -207,18 +209,25 @@ describe("Accept seller offer", () => {
       const submitButton = component.find(Button).last()
       submitButton.simulate("click")
 
-      const errorComponent = component.find(ErrorModal)
+      await flushPromiseQueue()
+      component.update()
+
+      const errorComponent = component.find(ModalDialog)
       expect(errorComponent.props().show).toBe(true)
       expect(errorComponent.text()).toContain("An error occurred")
       expect(errorComponent.text()).toContain(
         "Something went wrong. Please try again or contact orders@artsy.net."
       )
 
       component.find(ModalButton).simulate("click")
-      expect(component.find(ErrorModal).props().show).toBe(false)
+
```

Comment:
These bits (lines 223-227) are repeated throughout the tests - we 1. render the component, 2. verify the modal shows up, 3. click the button to dismiss the modal, then 4. verify that the modal is gone. Is there value in repeating steps 3&4 in almost every test, or would it make more sense to write a single test against `injectDialog` that verifies this instead?

Reply:
Yeah that would be good. Shall we do it as a follow up or would you prefer to have it taken care of in this PR?

Reply:
Follow-up would be great!